README.md

```diff
@@ -67,6 +67,86 @@ Calabash-android will install an instrumentation along with your app when execut
 
 Every time you test a new binary or use an upgraded version of calabash a new test server will be build. The test server is an intrumentation that will run along with your app on the device to execute the test.
 
+Using UIAutomator2
+------------------
+
+To benefit from UIAutomator2 test capabilities (like interacting with other apps or system elements), you must start the test server with:
+`start_test_server_in_background(with_uiautomator: true)`
+
+Then you will be able to use the command perform_action('*method*', *{parameters}*) which includes the following actions.
+
+| Method | Parameter | Function |
+|--------|----------|-----------|
+| uiautomator_touch_text | Text | Tap on element with text |
+| uiautomator_touch_partial_text | Text | Tap on element containing text |
+| pull_notification | none | Open the notification shutter |
+| clear_notifications | none | Clear notifications and close the shutter |
+| uiautomator_text_dump | none | Dumps text from elements in the UI hierarchy |
+| uiautomator_ui_dump | Text | Dumps elements in the UI hierarchy |
+| wifi | 'on' or 'off' | Switch device wifi on or off |
+| wait_for_idle_sync | none | Wait for idle state |
```

Comment:
@rajdeepv hey! Do you know how this method is supposed to be used? Getting an error while usage 
```
perform_action('wait_for_idle_sync')
2021-08-12 17:49:51 - Action: wait_for_idle_sync - Params: 
2021-08-12 17:49:51 - Result:'{"bonusInformation":["Exception stack trace:\njava.lang.ArrayIndexOutOfBoundsException: length=0; index=0\n\tat sh.calaba.instrumentationbackend.actions.device.WaitForIdleSync.execute(WaitForIdleSync.java:21)\n\tat sh.calaba.instrumentationbackend.Command.execute(Command.java:47)\n\tat sh.calaba.instrumentationbackend.actions.HttpServer.runCommand(HttpServer.java:787)\n\tat sh.calaba.instrumentationbackend.actions.HttpServer.serve(HttpServer.java:767)\n\tat sh.calaba.instrumentationbackend.actions.NanoHTTPD$HTTPSession.run(NanoHTTPD.java:487)\n\tat java.lang.Thread.run(Thread.java:919)\n"],"message":"length=0; index=0","success":false}'
```

Reply:
@JoeSSS it needs a timeout in seconds. eg:
`perform_action('wait_for_idle_sync', 2)`

Reply:
ah, cool! Thanks a lot!